README.md

```diff
@@ -496,7 +496,10 @@ We recommend querying using payment key hashes (`addr_vkh`) when possible (other
   Output
 
   ```js
-  // current supplies of given assets
-  Array<number>
+  {
+    // current supplies of given assets.
+    // entry for an asset is null if it is not found.
+    supplies: Array<number | null>
```

Comment:
@ozgrakkurt , let's update this API a bit to be compatible with how it works here: https://github.com/Emurgo/yoroi-graphql-migration-backend/blob/master/src/services/utxoSumForAddress.ts#L42

You can accept the list of asset IDs as an array of strings of format `{policy}.{name}` and then the result should be an object with keys and valuesm like:

```
{
  supplies: {
    {assetId1}: {value},
    {assetId2}: {value},
    {assetId3}: {value},
     ...
  }
}
```

If some asset ID is not found or recognised - it's just ignored and not included into the result.

Returning just the array might be too prone for potential bugs that would be hard to debug and investigate (e.g. the ordering) and will be harded to use from the frontend.

Reply:
Ah damn, while I was writing the comment @pedromtcosta alreadt got exactly same thing 😄

Reply:
fixed in 3f6c0bd2d9d325c7e06db0f3d853012c6a8d4d5f